manifests/base/generate_kustomization.py

```
@@ -0,0 +1,426 @@
+    # Fall back: find any resource containing the imagestream name
+    for res in all_resources:
+        if imagestream_name in res:
+            return res
+    # If not found, this is likely a buildconfig or similar non-imagestream resource
+    # Return empty and let the caller handle it
+    return ""
+
+
+def _order_resources(
+    all_resources: list[str],
+    workbench_files: list[str],
```

Ruff (0.15.0)

[warning] Unused function argument: `workbench_files`

(ARG001)